docs/content/configuration/global-configuration/globalconfiguration-resource.md

```diff
@@ -45,12 +53,15 @@ spec:
 
 ### Listener
 
-The listener defines a listener (a combination of a protocol and a port) that NGINX will use to accept traffic for a [TransportServer](/nginx-ingress-controller/configuration/transportserver-resource):
+The listener defines a listener (a combination of a protocol and a port) that NGINX will use to accept traffic for a [TransportServer](/nginx-ingress-controller/configuration/transportserver-resource) and a [VirtualServer](nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources):
```

Comment:
I feel this could be improved to make more sense. What is the first `listener` in this context?

Could it be reworded to "The listener property defines a listener", replace `property` with feature, entry, key, whatever makes sense in this context